data_validation/partition_builder.py

```diff
@@ -88,15 +113,14 @@ def _extract_where(table_expr, client) -> str:
         ).replace("t0.", "")
 
     def _get_partition_key_filters(self) -> List[List[List[str]]]:
-        """Generate where clauses for each partition for each table pair. We have to create separate where for the source and
-            target as they may each have a different filter applied to them. We are partitioning the tables based on keys, so that
-            we get equivalent sized partitions that can be compared against each other. With this approach, we can validate the
-            partitions in parallel leading to horizontal scaling of DVT. The design doc for this section is available in
+        """Given self.config_managers - i.e. a list of table pairs for row validation, generate the paritition filter
```

Comment:
nit: typo - misspelled 'partition filter'

Since you aren't inputting a ConfigManager object, it's confusing to say 'given self.config_managers'.